docs/stransmit.md

```diff
@@ -164,15 +164,21 @@ example, **-t:60**. Alternatively you can also separate them by a space,
+- **-timeout, -t, -to** - Sets the timeout for any activity from any medium (in seconds). Default is 0 for infinite (that is, turn this mechanism off). The mechanism is such that the SIGALRM is set up to be called after the given time and it's reset after every reading succeeded. When the alarm expires due to no reading activity in defined time, it will break the application. **Notes:**
     - The alarm is set up after the reading loop has started, **not when the application has started**. That is, a caller will still wait the standard timeout to connect, and a listener may wait infinitely until some peer connects; only after the connection is established is the alarm counting started. 
     - **The timeout mechanism doesn't work on Windows at all.** It behaves as if the timeout was set to **-1** and it's not modifiable.
+- **-timeout-mode, -tm** - timeout mode used. Default is 0 - timeout will happen after the specified time. Mode 1 cancels the timeout if the connection was established.
 - **-chunk, -c** - use given size of the buffer. When 0, it uses default 1316, which is the maximum size for a single SRT sending call
-- **-bandwidth, -bitrate, -b** - slow down sending to not send more data than given bandwidth (in bits per second). Note that it has nothing to do with SRT-controlled bandwidth; rather a possibility to have a "live-alike" reading speed from an infinite-speed input source (such as file). Better to think of it as for "testing purposes" because the real live transmission requires its speed synchronized with TS timestamps and *srt-live-transmit* has no ability to interpret them.
+- **-bandwidth, -bitrate, -b** [**To be added**] - slow down sending to not send more data than given bandwidth (in bits per second). Note that it has nothing to do with SRT-controlled bandwidth; rather a possibility to have a "live-alike" reading speed from an infinite-speed input source (such as file). Better to think of it as for "testing purposes" because the real live transmission requires its speed synchronized with TS timestamps and *srt-live-transmit* has no ability to interpret them.
 - **-verbose, -v** - display additional information on the standard output. Note that it's not allowed to be combined with output specified as **file://con**
-- **-bandwidth-report, -bitrate-report, -report, -r** - Throw periodic bandwidth reports on the standard output. The value is the number of sending events between reports.
-- **-crash, -k** - Force the program to "crash" (terminate abnormally with core dump) when a problem occurred (even such as connection got broken). This is useful only for developers.
+- **-bandwidth-report, -bitrate-report, -report, -bwreport, -r** - Show periodic bandwidth reports on the standard output. The value is the number of sending events between reports.
```

Comment:
AFAIR this option has been removed, or at least was planned to be removed. It doesn't make any sense because this slowdown doesn't come together with transport stream timestamps. It was for testing purposes only, but even in this form it's limited.

Reply:
Good. Deleted from the docs.